model_incoming_transactions.go

```diff
@@ -296,7 +294,7 @@ func processIncomingTransaction(ctx context.Context, logClient zLogger.GormLogge
 		ctx, incomingTx.ID, chainstate.RequiredInMempool, defaultQueryTxTimeout,
 	); err != nil {
 
-		logClient.Error(ctx, fmt.Sprintf("processIncomingTransaction(): error finding transaction %s on chain. Reason: %s", incomingTx.ID, err))
+		logClient.Error().Msgf("processIncomingTransaction(): error finding transaction %s on chain. Reason: %s", incomingTx.ID, err)
```

Comment:
Please add `txID` as context info

Reply:
You can add `processIncomingTransaction()` to `Str('processIncomingTransaction()')`

Reply:
Should I add some parameter like function name to every log? Or just to these ones which has the function name. I have to have key and value so it could be like `Str("function_name", "processIncomingTransaction()")` but we can have it from log_origin

Reply:
Removed